### DISCUSSION TITLE
SmartCardPlugin Icon

### DISCUSSION THREAD

**Discussion by ghost** [elrumo/macOS_Big_Sur_icons_replacements#1347] [Icon request] [answered]
This is the last app on my mac without a proper big sur icon, the default icon is really ugly but I don't use it often so it's fine, if you could size it to MacOS big sur icon size or try something original it would be great.

It's an app made to reload your metro transport card with your mac.

Here's the default icon:
![scp](https://user-images.githubusercontent.com/93690138/146603765-dc11e214-e36e-40d1-a5c6-ee81c9631a00.jpg)

**Comment by bunkerdesign** [comment, marked as answer]
[Smart Card Plugin.icns.zip](https://github.com/elrumo/macOS_Big_Sur_icons_replacements/files/7763664/Smart.Card.Plugin.icns.zip)
here you go

**Reply by ghost** [reply]
Thanks, I don't know how you did it but it's beautiful
